apps/site/next.dynamic.constants.mjs

```diff
@@ -29,6 +32,14 @@ export const IGNORED_ROUTES = [
  * @type {Map<string, import('./types').Layouts>} A Map of pathname and Layout Name
  */
 export const DYNAMIC_ROUTES = new Map([
+  // Creates dynamic routes for downloads archive pages for each version
+  // (e.g., /download/v18.20.8, /download/v20.19.2)
+  ...provideReleaseData()
+    .flatMap(({ minorVersions, versionWithPrefix }) => [
```

Comment:
Instead of a flatMap make two separate maps for readability, as we did for the blog categoreis and posts... Also instead of relying on provideReleaseData() maybe the providers could have a simple `provideReleaseVersions()` including all our versions.